Core/src/org/sleuthkit/autopsy/report/GeneralReportModule.java

```diff
@@ -33,13 +31,4 @@
      */
     public void generateReport(String baseReportDir, ReportProgressPanel progressPanel);
 
-    /**
-     * Returns the configuration panel for the report, which is displayed in the
-     * report configuration step of the report wizard.
-     *
-     * @return Configuration panel or null if the module does not need
-     *         configuration.
-     */
-    public JPanel getConfigurationPanel();
```

Comment:
I am noting that this is a public API change removing this method as the report package is exported.